terraform-dev/cloudinit.cfg

```diff
@@ -0,0 +1,44 @@
+#cloud-config
+packages: [python, curl, htop, iotop, lsof, ltrace, mc, net-tools, strace, tcpdump, telnet, vim, wget, ntp, traceroute, bash-completion]
+ssh_authorized_keys: ["${file("ssh/key.pub")}"]
+manage_resolv_conf: true
+resolv_conf:
+  nameservers: ['8.8.4.4', '8.8.8.8', '1.1.1.1']
+  options:
+    rotate: true
+bootcmd:
+- echo ${ip_address} ${hostname} >> /etc/hosts
+hostname: "${hostname}"
+write_files:
+- content: "br_netfilter"
```

Comment:
Most active releases should actually set and load modules, and if they don't it's an issue with the release. I would consider dropping the cloud-config that sets this, so that problems with the modules get surfaced. The downside is when trying to use an old/broken release it will keep popping up.

Reply:
Actually, I'm not sure if br_netfilter works correctly on 5.5, but I think the other modules should/do.

Reply:
I personally use terraform-dev a lot to test upgrades from 5.0.x and I need those modules to be enabled.
I believe we should test modules autoload in gravity PRs by disabling the same code in robotest terraform scripts. @wadells

Reply:
I filed https://github.com/gravitational/robotest/issues/184 to track testing our module autoload.